[jsk_spotkinova_robot] Update README.md

a-ichikura opened the pull request:
Add
・How to dock / undock spot
・When Ros cannot connect to Belka
・How to change pose with euslisp

@708yamaguchi

a-ichikura committed:
Update README.md

Add
・How to dock / undock spot
・When Ros cannot connect to Belka
・

a-ichikura committed:
[jsk_spotkinova_robot] Update README.md

Add
・How to dock / undock spot
・When Ros cannot connect to Belka
・How to change pose with euslisp
・caution to spotkinova-interface.l
・caution to hijack by the tablet

@708yamaguchi

@708yamaguchi

a-ichikura committed:
[jsk_spotkinova_robot] update README.md

Add
・how to use spotkinova's camera
・how to use coral usb with spotkinova's camera

708yamaguchi commented on jsk_spotkinova_robot/README.md:
````diff
@@ -229,3 +230,33 @@ You can use `inverse-kinematics` to move arm.
+*This process needs ssh*
+
+```bash
+ssh ichikura@belka.local
+[password]
+source /opt/ros/melodic/setup.bash
+source /home/spot/spot_coral_ws/devel/setup.bash
````
```
ssh spot@belka.local
[password]
source /opt/ros/melodic/setup.bash
source $HOME/spot_coral_ws/devel/setup.bash
```
?

a-ichikura replied:
Yes, it is spot@belka.local.

708yamaguchi commented on jsk_spotkinova_robot/README.md:
```diff
@@ -97,26 +158,33 @@ If you want to send an angle-vector to kinova, you need to send the joint angles
+(send *ri* :body-pose '(0 0 0.2))
+(send *ri* :body-pose '(0 0 -0.2))
+(send *ri* :body-pose '(0.1 0 0))
+(send *ri* :body-pose '(-0.1 0 0))
```
I am sorry, but could you please add information about coords?

Like

To change the full body pose, you can use `body-pose` method with rpy list or coords variable.
```
(send *ri* :body-pose '(0 0.2 0))
(send *ri* :body-pose '(0 -0.2 0))
(send *ri* :body-pose '(0 0 0.2))
(send *ri* :body-pose '(0 0 -0.2))
(send *ri* :body-pose '(0.1 0 0))
(send *ri* :body-pose '(-0.1 0 0))
(send *ri* :body-pose (make-coords :pos #f(0 0 50) :rpy #f(0.1 0 0)))
```

a-ichikura replied:
Ok, I will add it.

a-ichikura committed:
[jsk_spotkinova_robot] update README.md

fix typo in Coral USB

```bash
ssh spot@belka.local
[password]
source /opt/ros/melodic/setup.bash
source $HOME/spot_coral_ws/devel/setup.bash
```

a-ichikura committed:
[jsk_spotkinova_robot] update README.md

Add `make-coords` with `body-pose`.
```
(send *ri* :body-pose (make-coords :pos #f(0 0 50) :rpy #f(0.1 0 0)))
```

708yamaguchi reviewed (approved):
Thank you very much for updating minor changes.

mqcmd196 commented:
+1
